mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.test.js

```diff
@@ -229,4 +249,125 @@ describe('ExperimentRunsTableMultiColumnView2', () => {
     wrapper = shallow(<Router>{output}</Router>);
     expect(wrapper.html()).toContain('1 more');
   });
+
+  test('getColumnDefs should return only attribute columns that are not unchecked', () => {
+    const expectedColumnNames = [
+      undefined,
```

Comment:
Why is first column name undefined?

Reply:
The first column definition does not define a header name since its the "column" of the run checkboxes. See: https://github.com/mlflow/mlflow/blob/53bf22dac0de6311dec6c9ff288f46fc7170df41/mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js#L159-L167

Reply:
Ahhh yes, that makes sense. thanks for that info